.gitignore

```diff
@@ -0,0 +1,98 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This is an example comment. You can make these too by hovering around the line number and clicking [+]